WTWSMS/common/minor_titles/WtWSMS_minor_titles.txt

```diff
@@ -132,20 +132,21 @@ title_sigecempa = {
 }
 
 title_court_celtic_poet = { # Celtic Poet/Bard
-	dignity = 0.25
+	dignity = 1
```

Comment:
It doesn't need to show up though, as far as I know it is still calculated in the background.